### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by Spinoscythe**
What the title says

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,62 +4,62 @@ fabric.loom.multiProjectOptimisation=true
...
+fabric-resource-loader-v0-version=1.0.0
+fabric-screen-api-v1-version=3.0.0
+fabric-screen-handler-api-v1-version=2.0.0
+fabric-sound-api-v1-version=2.0.0
+fabric-transfer-api-v1-version=5.0.0
 fabric-transitive-access-wideners-v1-version=5.0.14
-fabric-convention-tags-v1-version=1.5.10
-fabric-client-tags-api-v1-version=1.1.7
+fabric-convention-tags-v1-version=2.0.0
+fabric-client-tags-api-v1-version=2.0.0
```
Comment (modmuss50) [inline gradle.properties]: Can you revert the changes here please, this is not right at all, and something I do when preparing a release.

File: gradle.properties [hunk 2]
```diff
@@ -4,62 +4,62 @@ fabric.loom.multiProjectOptimisation=true
 
 version=0.96.3
 minecraft_version=1.20.4
-yarn_version=+build.1
-loader_version=0.15.1
+yarn_version=+build.3
+loader_version=0.15.7
```
Comment (modmuss50) [inline gradle.properties:8]: This should be causing the build to fail, can you check that it builds.

**Review by modmuss50 — COMMENTED** [review]
Im not too sure about this, I generally only update the depedencies when needed.

There are other things that should also be checked for newer versions. (e.g checkstyle, github actions).

**Comment by Spinoscythe** [comment]
ok, noted
